index.html

```diff
@@ -1469,7 +1469,28 @@ <h4>@context</h4>
           </tbody>
         </table>
 
-
+        <p>
+The following values are acceptable values for <code>@context</code> property as
+a <a href="https://tools.ietf.org/html/rfc8259#section-7">JSON
+  String</a> or first item of a <a
+  href="https://tools.ietf.org/html/rfc8259#section-5">JSON Array</a>,
+represented as a <a href="https://tools.ietf.org/html/rfc8259#section-7">JSON
+  String</a>.
+        </p>
+        <table class="simple" style="width: 100%;">
+          <thead>
```

Comment:
We should have a link to the latest official version of the specification to go with each valid `@context` value.

Reply:
Will add this

Reply:
I am requesting changes on this, until notified this is done.